lib/xcodeproj/scheme/environment_variables.rb

```diff
@@ -0,0 +1,117 @@
+require 'xcodeproj/scheme/xml_element_wrapper'
+
+module Xcodeproj
+  class XCScheme
+    VARIABLES_NODE = 'EnvironmentVariables'
+    VARIABLE_NODE = 'EnvironmentVariable'
+
+    # This class wraps the EnvironmentVariables node of a .xcscheme XML file. This
+    # is just a container of EnvironmentVariable objects. It can either appear on a
+    # LaunchAction or TestAction scheme group.
+    #
+    class EnvironmentVariables < XMLElementWrapper
+      # @param [REXML::Element,Array{EnvironmentVariable},Array{Hash{String => String}}] node_or_target
+      #   The 'EnvironmentVariables' XML node that this object represents.
```

Comment:
After reading the rest of the code and realizing that a EnvVariable is not just a key & value but also enabled, it seems OK to keep that. (Also please keep the keys as symbols like you did).

The only thing missing then is proper documentation to tell how this Hash is supposed to be organized and what key symbols are expected.